consensus/p2p/validator/fixtures_test.go

```diff
@@ -144,8 +172,14 @@ func buildProposalInit(
 	}
 }
 
-func buildBlockInfo(height types.Height, proposer *common.Address, block *core.Block) consensus.ProposalPart {
-	ethToStrkRate := new(felt.Felt).Div(block.L1DataGasPrice.PriceInFri, block.L1DataGasPrice.PriceInWei)
+func buildBlockInfo(
+	height types.Height,
+	proposer *common.Address,
+	block *core.Block,
+) consensus.ProposalPart {
+	ethToStrkRate := new(
+		felt.Felt,
+	).Div(block.L1DataGasPrice.PriceInFri, block.L1DataGasPrice.PriceInWei)
```

Comment:
Personal opinion: This looks better to me:
```suggestion
	ethToStrkRate := new(felt.Felt).Div(
		block.L1DataGasPrice.PriceInFri,
		block.L1DataGasPrice.PriceInWei,
	)
```